src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+    static String getRandomColor() {
```

Comment:
```suggestion
    public String getRandomColor() {
```
Don't change signature of method

Reply:
If i change it and remove "static" it will cause an error
![2024-04-24 09_41_25-jv-lottery – ColorSupplier java](https://github.com/mate-academy/jv-lottery/assets/146541661/c76e2222-b7a6-4e80-8c98-856e4a6998f3)
![2024-04-24 09_41_39-jv-lottery – Lottery java](https://github.com/mate-academy/jv-lottery/assets/146541661/3953472f-14db-428b-b7f4-e1731db06189)

Reply:
Can i leave that method "public static String"?